src/jdk.compiler/share/classes/com/sun/tools/javac/resources/javac.properties

```diff
@@ -206,6 +206,9 @@ javac.opt.Xlint.desc.fallthrough=\
 javac.opt.Xlint.desc.finally=\
     Warn about finally clauses that do not terminate normally.
 
+javac.opt.Xlint.desc.lossy-conversions=\
+    Warn about compiler possible lossy conversions.
```

Comment:
I like this warning.  But the documentation doesn't even parse as English.

I suggest this, as more grammatical and precise:

> Warn about possible lossy conversions in compound assignments